### PR TITLE
Update the UnitTests project to use xUnit.net

### PR DESCRIPTION
**Pull request by sharwell**
Update the UnitTests project to use xUnit.net

Following this change, tests in the project can be run without special configuration (e.g. selection of the testsettings file).

### CODE REVIEW

File: Foxtrot/Tests/UnitTests/GenericsTest.cs [hunk 1]
```diff
@@ -15,286 +15,231 @@
...
+using Xunit;
 
 namespace Tests {
   /// <summary>
   /// Summary description for GenericsTest
   /// </summary>
-  [TestClass]
-  public class GenericsTest : DisableAssertUI {
-    public GenericsTest()
+  public class GenericsTest : DisableAssertUI, IClassFixture<GenericsTest.GenericsTestFixture> {
```
Comment (SergeyTeplyakov) [inline Foxtrot/Tests/UnitTests/GenericsTest.cs]: Maybe it would be easier to format this stuff first?

Comment (sharwell) [inline Foxtrot/Tests/UnitTests/GenericsTest.cs]: :question: Did I mess up the formatting? I tried to match the formatting already in place.

File: Foxtrot/Tests/UnitTests/UnitTests.csproj [hunk 2]
```diff
@@ -95,12 +80,21 @@
...
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.VisualStudio.QualityTools.UnitTestFramework, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
     <Reference Include="System" />
-    <Reference Include="System.Core">
-      <RequiredTargetFramework>3.5</RequiredTargetFramework>
-    </Reference>
+    <Reference Include="System.Core" />
     <Reference Include="System.Windows.Forms" />
+    <Reference Include="xunit.abstractions, Version=2.0.0.0, Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c, processorArchitecture=MSIL">
```
Comment (SergeyTeplyakov) [inline Foxtrot/Tests/UnitTests/UnitTests.csproj]: Is it correct that xunit.abstractions is of version 2.0.0.0 and the rest of the xunit packages are 2.1.0-beta4?

**Comment by SergeyTeplyakov** [comment]
Everything looks good.

File: Foxtrot/Tests/UnitTests/UnitTests.csproj (hunk 2 above)
Comment (sharwell) [inline Foxtrot/Tests/UnitTests/UnitTests.csproj]: Yes that's correct :+1:

File: Foxtrot/Tests/UnitTests/GenericsTest.cs (hunk 1 above)
Comment (SergeyTeplyakov) [inline Foxtrot/Tests/UnitTests/GenericsTest.cs]: No, you're not:) I just was surprised that we didn't reformat this code yet:)

File: Foxtrot/Tests/UnitTests/UnitTests.csproj (hunk 2 above)
Comment (SergeyTeplyakov) [inline Foxtrot/Tests/UnitTests/UnitTests.csproj]: Ok. Understood.

**Comment by SergeyTeplyakov** [comment]
Awesome! Merging...

**Pull request merged**